projectmanagement/src/main/java/life/qbic/projectmanagement/domain/ProjectRepository.java

```diff
@@ -0,0 +1,13 @@
+package life.qbic.projectmanagement.domain;
+
+/**
+ * <b>short description</b>
+ *
+ * <p>detailed description</p>
+ *
+ * @since <version tag>
+ */
+public interface ProjectRepository {
+
+  void add(Project project);
```

Comment:
Missing JD

Reply:
This will be added by the project repository PR by @wow-such-code 
I only defined the interface here to be replaced once the other branch is merged